parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -85,6 +86,10 @@ if [[ $OS == $UBUNTU_OS_NAME ]]; then
     time_metric "EnsureAuditD" ensureAuditD
 fi
 
+if [[ "$FULL_INSTALL_REQUIRED" = "true" ]]; then
+    time_metric "InstallBpftrace" installBpftrace "v0.9.4"
```

Comment:
does this work with non-ubuntu distros, eg. coreos?

Reply:
yes

Reply:
it will work on any linux kernel with glibc > 2.23 compile with BPF enabled (i.e., very broad)